Scripts/RemediationScripts/Remediate-SecureFTPDeploymentForAppServices.ps1

```diff
@@ -495,7 +522,7 @@ function Enable-SecureFTPDeploymentForAppServices
                 return
```

Comment:
Can you explain else block from line 509?

Reply:
Closed as per discussion in the call.